terraform/environments/staging/variables.tf

```diff
@@ -27,7 +32,26 @@ variable "session_bucket_ttl_days" {
   default     = 14
 }
 
-variable "ops_project_id" {
+variable "setup_cd_system" {
+  type        = bool
+  default     = false
+  description = "Create deployment triggers. Enable only if Cloud Build has GitHub repository access."
+}
+
+variable "repo_owner" {
+  description = "Repo Owner (GoogleCloudPlatform)"
+  type        = string
+  default     = ""
+}
+
+variable "repo_name" {
+  description = "Repo Name (emblem)"
+  type        = string
+  default     = ""
```

Comment:
I left this unchanged from earlier implementation. We could set emblem default because GitHub forks will use it but I think that's fair for a follow-up.